frontend/src/app/schema/common.js

```diff
@@ -65,6 +65,12 @@ export const port = {
     maximum: 65535
 };
 
+export const multipart = {
```

Comment:
this should be defined in `src/api/common_api.js` I would also change the name to something clearer like `multipart_count` or something similar